src/main/java/com/marek/onlinebookstore/controller/BookController.java

```diff
@@ -0,0 +1,36 @@
+package com.marek.onlinebookstore.controller;
+
+import com.marek.onlinebookstore.dto.BookDto;
+import com.marek.onlinebookstore.dto.CreateBookRequestDto;
+import com.marek.onlinebookstore.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/books")
+public class BookController {
+    private BookService bookService;
```

Comment:
```suggestion
    private final BookService bookService;
```

Reply:
Variable 'bookService' might not have been initialized ?